src/common/auth/jwt_parser.rs

```diff
@@ -59,22 +62,22 @@ mod tests {
             .as_secs();
         let claims = Claims {
             exp: Some(exp),
-            w: Some(true),
-            access: Access {
-                collections: Some(vec!["collection".to_string()]),
-                payload: Some(
+            access: Access::Collection(AccessCollection(vec![AccessCollectionRule {
+                collections: vec!["collection".to_string()],
+                access: CollectionAccessMode::ReadWrite,
+                payload: Some(PayloadConstraint(
                     vec![
                         (
                             "field1".parse().unwrap(),
                             ValueVariants::Keyword("value".to_string()),
                         ),
                         ("field2".parse().unwrap(), ValueVariants::Integer(42)),
-                        ("field2".parse().unwrap(), ValueVariants::Bool(true)),
+                        ("field3".parse().unwrap(), ValueVariants::Bool(true)),
                     ]
                     .into_iter()
                     .collect(),
-                ),
-            },
+                )),
+            }])),
             value_exists: None,
         };
```

Comment:
we don't yet, but it might be a _nice to have_

Reply:
I'll think about it when adding tests. For now, I'd keep it as is since it's only used in a single test, and it's not supposed to be created outside of `serde::Deserealize` and tests.

Reply:
Yeah. Since we only use it here; fine by me!